nulib/next-gen-front-end-react
# Bump react-media-player out of alpha.

## Summary 
Update for @nulib/react-media-player package.

## Specific Changes in this PR
- Bumps **@nulib/react-media-player** from `1.0.0-alpha4` to `1.0.1`

## Steps to Test

1. `npm install`
2. `npm run start:use-staging-data`
3. Navigate to non-Image work in browser, react-media-player should be most current version (purple header issue resolve, actual vtt from manifests parsed)
